winton_kafka_streams/kafka_config.py

```diff
@@ -239,6 +234,53 @@
 """
 MAX_POLL_RECORDS = 1000
 
+#### - Serdes Configuration - ####
```

Comment:
No strong feelings here either way, this could also be done with a DoubleSerializer and a FloatSerializer instead of just one with extra config.

Thoughts?

Reply:
Yes, that's probably sensible. Will declutter the config if nothing else.